go.mod

```diff
@@ -3,7 +3,7 @@ module helm.sh/helm/v3
 go 1.14
 
 require (
-	github.com/BurntSushi/toml v0.3.1
+	github.com/BurntSushi/toml v0.3.1 // indirect
```

Comment:
It looks like the legacy library is still in use somewhere in the dependency tree. Do you happen to know what's using it?

Reply:
There are still many projects using `BurntSushi/toml`, like containerd that was also thinking of changing https://github.com/containerd/containerd/issues/4370.